### ISSUE TITLE
span masking

### ISSUE THREAD

**Issue by jianglong-he-Infrrd** [uakarsh/latr#12]
Hi, in `apply_mask_on_token_bbox` method it looks like you are masking only first token of the span then loose the bbox of whole span. Why not masking all tokens in the span?

**Comment by uakarsh** [comment]
Yep that is right, I should have done that earlier, but I was not knowing that earlier. Am just reflecting that from sometime back, so would do that soon.
